aspnetcore/security/blazor/index.md

```diff
@@ -24,6 +24,8 @@ Security scenarios differ between Blazor Server and Blazor WebAssembly apps. Bec
 
 Blazor WebAssembly apps run on the client. Authorization is *only* used to determine which UI options to show. Since client-side checks can be modified or bypassed by a user, a Blazor WebAssembly app can't enforce authorization access rules.
 
+[Razor Pages authorization conventions](xref:security/authorization/razor-pages-authorization) don't apply to routable Razor components. If an unroutable Razor component is [embedded in a page](xref:blazor/integrate-components#render-components-from-a-page-or-view), the Razor component is indirectly effected by the page's authorization conventions along with the rest of the page's content.
```

Comment:
`effected` or `affected`?

Reply:
*Yes* ... "affected" for **_impact on_**. Good :eye:!